### PR TITLE
Deprecate ClassDescription>>#addMethodTag: 

### PR DESCRIPTION
**Pull request by jecisc**
This method just calls #addProtocol: and protocol is the name currently used in the system.

### CODE REVIEW

File: src/Ring-Tests-Core/RGBehaviorTest.class.st [hunk 1]
```diff
@@ -18,13 +18,14 @@ RGBehaviorTest >> testBadInstantiation [
...
+
 	| newBehavior |
 	newBehavior := self behaviorClass named: #SomeClass.
 	self assert: newBehavior name equals: #SomeClass.
-	self assert: (newBehavior hasUnresolvedAll: #(superclass localMethods protocols)).
+	self assert: (newBehavior hasUnresolvedAll: #( superclass localMethods protocols )).
 	self assert: newBehavior methods isEmpty.
 
-	1 to: 10 do: [ :i | newBehavior addMethodTag: ('tag' , i asString) asSymbol ].
+	1 to: 10 do: [ :i | newBehavior addProtocol: ('tag' , i asString) asSymbol ].
```
Comment (demarey) [inline src/Ring-Tests-Core/RGBehaviorTest.class.st:28]: better to replace 'tag' by 'protocol' too

File: src/Ring-Tests-Core/RGBehaviorTest.class.st [hunk 2]
```diff
@@ -39,13 +40,14 @@ RGBehaviorTest >> testBehaviorWithMethodTags [
...
```
Comment (demarey) [inline src/Ring-Tests-Core/RGBehaviorTest.class.st:50]: better to replace 'tag' by 'protocol' too

File: src/Ring-Tests-Core/RGClassTest.class.st [hunk 3]
```diff
@@ -339,25 +339,24 @@ RGClassTest >> testTagsForMethods [
 RGClassTest >> testTagsForMethodsCollection [
 
 	| class env tag1 tag2 tag3 |
```
Comment (demarey) [inline src/Ring-Tests-Core/RGClassTest.class.st:341]: better to replace tagx by protocolx

**Review by demarey — APPROVED** [review]

**Comment by jecisc** [comment]
@demarey I'm planning to do that but when I start to replace every "tag" I see, I end up with PR of hundreds of lines of code so I'm going by step for now :)

Between the methods, the parameters, the temporaries, the class names... There are a lot of things to rename.

**Comment by jecisc** [comment]
Test failure is due to Spec and will soon be fixed (I'll do a PR in a few minutes)

**Pull request merged**